invenio_theme/assets/semantic-ui/less/invenio_theme/theme/globals/site.overrides

```diff
@@ -199,6 +199,16 @@ code {
   }
 }
 
+pre {
+  font-family: inherit;
+  font-size: inherit;
+  margin: 0;
+}
```

Comment:
This would affect all `pre` tags. 
Also I thing this can be removed as the PR that would need this was updated and doesn't have the `pre` tag anymore, see [here](https://github.com/inveniosoftware/invenio-administration/pull/162) the PR.
```suggestion

```